javaparser-core/src/main/java/com/github/javaparser/ast/nodeTypes/NodeWithName.java

```diff
@@ -40,7 +38,7 @@
 
     @SuppressWarnings("unchecked")
     default N setName(String name) {
-        return setName(parse(name));
+        return setName(Name.parse(name));
```

Comment:
Well, I found both ways to use the parse method in source code, so I decided to also use my preferred way where you immediately see where the method comes from :D